packages/botonic-plugin-contentful/src/contentful/manage/manage-entry.ts

```diff
@@ -181,25 +174,27 @@ export class ManageContentfulEntry {
   }
 
   private convertValueType(key: ContentFieldType, field: any): any {
+    const valueType = CONTENT_FIELDS.get(key)?.valueType
+    if (field === undefined || valueType === typeof field) return field
     if (key === ContentFieldType.BUTTONS_STYLE) {
-      return field ? 'QuickReplies' : 'Buttons'
+      return field === ButtonStyle.QUICK_REPLY ? QUICK_REPLY : BUTTON
```

Comment:
review my latest comment. you need to store "null" when this field is undefined.
Since you you're now supporting yet this case in TextFields, please create a ticket to support in a different PR.